wayou/vscode-todo-highlight
# Adding review highlight and listing review

Overview:
This pull request introduces enhancements to the REVIEW annotation features within our VSCode extension. The primary goal is to improve the visibility and management of REVIEW tags in the codebase, facilitating easier and more effective code reviews.

Motivation:
Following our discussion in issue #243 and feedback from several team members, it became evident that our current handling of REVIEW annotations was insufficient for the needs of our large-scale projects. These changes aim to address this gap and align with our goals for maintaining high code quality.

Changes Made:

Highlighting REVIEW Annotations: Added distinct purple background color and overview ruler markings for REVIEW annotations.
Listing Annotations Command: Implemented a new command to list all REVIEW annotations across the project, enhancing navigability.
User Configuration Options: Introduced settings allowing users to customize the appearance and behavior of REVIEW annotation highlights.
How to Test the Changes:

Checkout the branch feature/review-enhancements.
Open any project with existing REVIEW annotations.
Go to settings and ensure the "Highlight REVIEW Annotations" feature is turned on.
Verify that REVIEW annotations are highlighted as specified, and use the "List REVIEW Annotations" command from the command palette.


Additional Notes:
These changes are designed to be backward compatible with existing settings. However, extensive testing in different environments

https://github.com/wayou/vscode-todo-highlight/assets/97735495/7a65a5c7-6b42-45ea-8ed1-1a5a2ee79a1b


https://github.com/wayou/vscode-todo-highlight/assets/97735495/87e4953e-d98b-471e-a048-299ff59c8bd6



Request for Feedback:
I would appreciate feedback on the UI changes, especially the color choices and the visibility of the annotation list in varied lighting conditions. Any suggestions for further improvement or potential issues are also welcome.
